Exiled.CustomRoles/API/Extensions.cs

```diff
@@ -0,0 +1,68 @@
+// -----------------------------------------------------------------------
+// <copyright file="Extensions.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.CustomRoles.API
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+
+    using Exiled.API.Features;
+    using Exiled.CustomRoles.API.Features;
+
+    using UnityEngine;
+
+    /// <summary>
+    /// A collection of API methods.
+    /// </summary>
+    public static class Extensions
+    {
+        /// <summary>
+        /// Gets a <see cref="ReadOnlyCollection{T}"/> of the player's current custom roles.
+        /// </summary>
+        /// <param name="player">The <see cref="Player"/> to check for roles.</param>
+        /// <returns>A <see cref="ReadOnlyCollection{T}"/> of all current custom roles.</returns>
+        public static ReadOnlyCollection<CustomRole> GetCustomRoles(this Player player)
+        {
+            List<CustomRole> roles = new List<CustomRole>();
+
+            foreach (CustomRole customRole in CustomRole.Registered)
+            {
+                if (customRole.Check(player))
+                    roles.Add(customRole);
+            }
+
+            return roles.AsReadOnly();
+        }
+
+        /// <summary>
+        /// Registers an <see cref="IEnumerable{T}"/> of <see cref="CustomRole"/>s.
+        /// </summary>
+        /// <param name="customRoles"><see cref="CustomRole"/>s to be registered.</param>
+        public static void Register(this IEnumerable<CustomRole> customRoles)
```

Comment:
Might be nice to have a params option for this. Something like
```
Register(customRole1, customRole2);
```

Reply:
I'm not sure what you mean by this, please explain? Each CustomRole itself already has a TryRegister() method, this just makes life a tad easier if you have multiple roles, you can create the new classes and add them to a list, then use this extension to register them all at once.